src/model/enums/owner.ts

```diff
@@ -0,0 +1,4 @@
+export enum Owner {
```

Comment:
Do we need to be more specific in what kind of owner?

Reply:
Isn't that what `Owner.SYSTEM` and `Owner.EXTERNAL` does?

Reply:
To a degree. What can this owner own? Is it the entire system, a rundown or an action taken in the system?
We don't have to change it, but just be aware that the term is a bit ambiguous.

Reply:
The Owner owns whatever the Owner is placed on. 
So currently, Owner can be place on a RundownCursor, so the Owner owns the RundownCursor. 
This enum is to tell _who_ owns it, not _what_ it owns.